GET_STARTED.md

```diff
@@ -0,0 +1,390 @@
+
+- Linux Machine
+    - Support both ccf network types [virtual - release (TEE hardware)]
+    - Governance steps required to deploy your app, initialize, and start the network
+
+#### Testing: Using Sandbox.sh
+
+By runing sandbox.sh script, it is automatically starting a CCF network and deploy your application on it, the app is up and ready to receive calls,
+all the governance work is done for you.
+
+Start in a CCF Network in Release mode
+```bash
+/opt/ccf/bin/sandbox.sh --js-app-bundle ./js/dist/  --enclave-type release -p /opt/ccf/lib/libjs.enclave.so.signed
+...
+[12:00:00.000] Press Ctrl+C to shutdown the network
+# It is then possible to interact with the service
+```
+
+Start in a CCF Network in Virtual mode (the default mode for testing)
+```bash
+/opt/ccf/bin/sandbox.sh --js-app-bundle ./js/dist/
+...
+[12:00:00.000] Press Ctrl+C to shutdown the network
+# It is then possible to interact with the service
+```
+
+#### Testing: Using docker containers
+
+Build and run one of these docker files ["ccf_app_js.virtual" or "ccf_app_js.enclave"] to start a CCF network with one node and one member
+after that you need to execute governance steps to deploy the application and open the network for users to begin access the app endpoints.
+all the governance steps is done manually using [proposal submit and vote process](https://microsoft.github.io/CCF/main/governance/proposals.html).
+
+##### Build and run docker container to start a CCF network
+
+Start in a CCF Network in Release mode, via docker container based on config file "./config/cchost_config_enclave_js.json"
+
+```bash
+ docker build -t ccf-app-samples:js-enclave -f docker/ccf_app_js.enclave .
+ docker run -d --device /dev/sgx_enclave:/dev/sgx_enclave --device /dev/sgx_provision:/dev/sgx_provision -v /dev/sgx:/dev/sgx ccf-app-samples:js-enclave
+ ...
+ # CCF Network initialization needed before the interaction with the service
+```
+
+Start in a CCF Network in Virtual mode, based on virtual config file: "./config/cchost_config_virtual_js.json":
+```bash
+ docker build -t ccf-app-samples:js-virtual -f docker/ccf_app_js.virtual .
+ docker run -d ccf-app-samples:js-virtual
+ ...
+ # CCF Network initialization needed before the interaction with the service
+```
+
+##### CCF Node Configuration file
+The configuration for each CCF node must be contained in a single JSON configuration file like [cchost_config_enclave_js.json - cchost_config_virtual_js.json], [ CCF node config file documentation](https://microsoft.github.io/CCF/main/operations/configuration.html)
+
+##### CCF network initialization
+After the container run, a network is started with one (node - member), you need to execute the following governance steps to initialize the network, [check Network governance section](#network-governance)
+- Activate the network members (to begin network governance)
+- Add users (using proposal)
+- Deploy the application (using proposal)
+- Open the network for users (using proposal)
```

Comment:
Let's double check this later, but if I'm not mistaken the network must be opened before the application deployment.

Reply:
No, it is not needed